doctr/transforms/modules/pytorch.py

```diff
@@ -53,3 +53,15 @@ def __repr__(self) -> str:
         if self.preserve_aspect_ratio:
             _repr += f", preserve_aspect_ratio={self.preserve_aspect_ratio}, symmetric_pad={self.symmetric_pad}"
         return f"{self.__class__.__name__}({_repr})"
+
+
+class RandomGaussianNoise():
+    def __init__(self, mean=0.5, std=1.5):
+        self.std = std
+        self.mean = mean
+
+    def __call__(self, tensor):
+        return tensor + torch.randn(tensor.size()) * self.std + self.mean
+
+    def __repr__(self) -> str:
+        return self.__class__.__name__ + f"mean = {self.mean}, std = {self.std}"
```

Comment:
since we use modern python, don't use `+` between string, use f-string all the way :+1: 

I suggest inheriting from `doctr.utils.repr.NestedObject` so that you only have to modify the `def extra_repr(self) -> str:`